### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by AmitKlohan**
My first project using linters

### CODE REVIEW

**Review by OmarRamoun — COMMENTED** [review]
# 🟢 Project Approved! 🏆

## Hi @AmitKlohan 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

> **Note**: One last thing. Kindly, next time don't merge the pull request before you get the project approved. Thank you @AmitKlohan and have an awesome day

Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmitKlohan** [comment]
Thanks Ramoun for the boosting my confidence, I've noted down your note, will take care in future.

**Comment by AmitKlohan** [comment]
Thanks Ramoun for  boosting my confidence, I've noted down your note, will take care in future.

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+
+## Author
+
+👤 Amit Kumar
+
+- GitHub: [@AmitKlohan](https://github.com/AmitKlohan)
+
+
+
```
Comment (degisew) [inline README.md:21]: Please try to add more personal information like your LinkedIn profile, Twitter username, and also your location.

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> project on config linters
+
+
```
Comment (degisew) [inline README.md:7]: Please try to add the project description here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1 class="header">Hello Microverse</h1>
+</body>
+
+</html>
```
Comment (degisew) [inline index.html:16]: Please be sure that you put blank line for reason.

File: README.md [hunk 4]
```diff
@@ -1 +1,40 @@
...
+## Built With
+
+- HTML - 5
+- CSS - 3
+
```
Comment (teke85) [inline README.md:12]: Hi, @AmitKlohan Good Job including a well-documented ReadMe File. I suggest you also include **Package Manager: Node, NPM** to the built with section of your ReadMe File.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1 class="header">Hello Microverse</h1>
```
Comment (scientistsagoemarcus) [inline index.html:13]: Your project is complete! There is nothing else to say other than to work hard towards your dreams.
Congratulations! 🎉

**Review by degisew — COMMENTED** [review]
Hi @AmitKlohan 

Great job so far 👏 👏 

you are almost there. but still there are some issues needs to change for your next project

**### status: Required changes ♻️** 
 check the comments under the review.

**Review by scientistsagoemarcus — COMMENTED** [review]
Great job, congratulation once again.

**Pull request merged**